component/notify/watch.go

```diff
@@ -48,16 +50,36 @@ func Watch(ctx mongo.SessionContext, messages chan interface{}) error {
 
 						if n.Matches(instanceName, t, change) {
 
-							subject := fmt.Sprintf("%s: %s %s %s (%s)", n.Name, application.Name, instanceName, inst.FormatVersion(), displayStatus(t, change))
+							status := displayStatus(t, change)
+							portalURL := fmt.Sprintf("%s/apps/%s/instance/%s", cfg.Get["URL"], application.Name, instanceName)
 
-							body := fmt.Sprintf("%s\n\n %s/apps/%s/instance/%s", subject, cfg.Get["URL"], application.Name, instanceName)
-							if inst.CurrentState.Error != nil {
-								body = fmt.Sprintf("%s \n\n %s", body, inst.CurrentState.Error)
-							}
+							switch {
```

Comment:
If we remove the switch case and keep just two individual if statement, the user could use both workflows at once if they choose so, right?

Wouldn't that be better?

Reply:
The UI does not allow for users to provide more than 1 endpoint per notice setup. Additionally, it may be better to enforce users to setup a different notice per endpoint so they can be managed/enabled separate of one another (e.g. I may still want my SNS workflow to happen but disable Slack notifications if I'm getting spammed). Switch statement will help us add more cases/endpoints in the future.

Reply:
that makes 100% sense